src/components/JumpNRun/PooComponent.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="poo" @click="collectPoo" :style="{ left: `${props.positionX}px` }">
+    <img src="@/assets/jumpNrun/poo.png" alt="Poo" class="image" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue'
+
+const props = defineProps({
```

Comment:
Please use TS to define the props

defineProps<{id: numberr, positioX: number}>()